packages/@snowpack/plugin-babel/plugin.js

```diff
@@ -1,10 +1,12 @@
-const babel = require("@babel/core");
+const babel = require('@babel/core');
 
-module.exports = function plugin(config, options) {
+module.exports = function plugin() {
   return {
-    defaultBuildScript: "build:js,jsx,ts,tsx",
-    async build({ contents, filePath, fileContents }) {
-      const result = await babel.transformAsync(contents || fileContents, {
+    name: '@snowpack/plugin-babel',
+    async transform({contents, fileExt, filePath}) {
```

Comment:
We say in the docs that the Babel plugin replaces esbuild as your build plugin (ex: if you have JSX in your `.js` file). In that case, use `resolve: {input: ['.js', '.jsx', '.ts', '.tsx', '.mjs'], output: ['.js']}` + `load()`